### PR TITLE
#2 - readme and rultor config

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #2 
Added rultor config and updated readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+## Artipie management API
+
+This library contains management API and dashboard for Artipie. REST API allows to 
+
+- create repositories
+- manage users
+- manage repository permission
+- list repository items
+
+For more details check [this](https://github.com/artipie/artipie/blob/master/REST_API.md) page.
```
[the remark this answers is not in the document]
Comment (olenagerasimova) [reply README.md:23]: @olegmoz ok, we keep as a separate file (as it is now) or copy the whole description into this readme?

[the remark this answers is not in the document]
Comment (olegmoz) [reply README.md:23]: @olenagerasimova I think it will be better to keep separate file

**Comment by olenagerasimova** [comment]
@olegmoz thanks, done, I've also created issue https://github.com/artipie/artipie/issues/761 to not forget to remove `REST_API.md` from artipie.

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Pull request merged**